_posts/2020-11-24-Multiple-Audio-Interfaces.md

```diff
@@ -19,7 +19,7 @@ Under Windows, the best option to run multiple interfaces through Jamulus at the
 **Steps:**
 1. [Install and configure JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html) - Make sure you select your best audio interface in the configuration steps, also add `-r 48000 -p <desired frame buffer size>` to ensure JACK is outputting audio in the required sample rate for Jamulus (48.000Hz). If your frame buffer size is too low for your hardware, you'll need to change it to a higher value;
 2. Install and open Voicemeeter;
-3. Open configured Jack PortAudio shortcut, Jack Control and Jamulus software;
+3. Open configured JACK PortAudio shortcut, Jack Control and Jamulus software;
```

Comment:
So what about Jack Control? Is this not all caps?

Reply:
As far as I know, this is the correct spelling (see https://jackaudio.org/faq/jack_on_windows.html ).